config/config.go

```diff
@@ -1509,6 +1509,17 @@ var (
 	// and label name characters that are not alphanumerics/underscores to underscores.
 	// Unit and type suffixes may be appended to metric names, according to certain rules.
 	UnderscoreEscapingWithSuffixes translationStrategyOption = "UnderscoreEscapingWithSuffixes"
+	// NoTranslation (EXPERIMENTAL): disables all translation of incoming metric
+	// and label names.
+	//
+	// Note that because metrics in Open Telemetry are considered
+	// distinct if they share the same name but have different Type or Units, for
+	// instance "foo.bar" with units Seconds is a separate series from "foo.bar"
+	// with units Milliseconds. Because prometheus can't differentiate timeseries
```

Comment:
Unit in this context is singular isn'it it?
```suggestion
	// distinct if they share the same name but have different Type or Unit, for
	// instance "foo.bar" with unit Seconds is a separate series from "foo.bar"
	// with unit Milliseconds. Because prometheus can't differentiate timeseries
```